### PR TITLE
[loss] Added missing weight decay loss

### PR DESCRIPTION
**Pull request by kparichay**
Added missing weight decay loss into final loss for intermediate layers

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: nntrainer/src/fc_layer.cpp [hunk 1]
```diff
@@ -132,6 +132,10 @@ Tensor FullyConnectedLayer::forwarding(Tensor in, int &status) {
   } else {
     return hidden.apply(activation);
   }
+
+  if (weight_decay.type == WeightDecayType::l2norm) {
+    loss = weight_decay.lambda * 0.5f * (weight.l2norm());
+  }
 }
```
Comment (jijoongmoon) [inline nntrainer/src/fc_layer.cpp:139]: LGTM!
